Amazon/Marketplace/v7153/templates/jfrog-artifactory-ec2-instance.template.yaml

```diff
@@ -0,0 +1,408 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
On this whole file with params you need to put some bounds on all of them along with some descriptions. Like, `ArtifactoryProduct` - what should be here? I haven't a clue.